documentatie/use-cases/gsb-invoer-eerste-zitting.md

```diff
@@ -178,7 +187,7 @@ __Uitbreidingen:__
 &emsp; 6b2. De applicatie laadt de pagina voor het selecteren van een stembureau.
 
 ### Open punten
-- De use case beschrijft de oude modellen, met daarin alleen de vraag "Is er herteld?" In de nieuwe modellen zijn er drie vragen.
+- De use case beschrijft de oude modellen, met daarin alleen de vraag "Is er herteld?" In de nieuwe modellen zijn er drie vragen. => hoe ver willen we gaan met specificeren?
```

Comment:
In deze use case staat "De invoerder voert in of er herteld is.", dus het open punt is correct. Ik stel voor dat we de use case wat minder specifiek maken, bijv.: "De invoerder neemt het proces-verbaal in de applicatie over.", zodat we niet elk invoerscherm hoeven te beschrijven.